### PR TITLE
Record initial actions by the board without a meeting

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Nothing earth-shattering here, but it's good to do for transparency and others may find it interesting / helpful.

Going forward, these will be updated much more promptly after each decision: it's much easier with a structure in place.

### CODE REVIEW

File: content/foundation/meeting-minutes/actions-without-meeting/index.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+**Proposal:** All in favor of Project Lead (me once we hire me), Staff Engineer (soon to be Alice + future maintainer hires), and Principal for SME hires?
+
+**Proposed by:** Carter Anderson
+
+**Votes:** Unanamious yes
+
+## 2024-04-18: Usage of the Bevy logo by the unofficial Bevy Playfround
+
+![Usage of the Bevy logo by the unofficial Bevy Playfround](bevy_playground_logo_usage.png)
```
Comment (ItsDoot) [inline content/foundation/meeting-minutes/actions-without-meeting/index.md:44]:
```suggestion
![Usage of the Bevy logo by the unofficial Bevy Playground](bevy_playground_logo_usage.png)
```

**Comment by alice-i-cecile** [comment]
FYI I'm going to wait on 3 approvals by board members here to merge.

File: content/foundation/bylaws/index.md [hunk 2]
```diff
@@ -101,7 +101,9 @@ A Director of the Corporation present at a Board meeting at which action on any
...
+Any action which could be taken at a meeting of the Board may be taken without a meeting if a written consent setting forth the action so taken is executed by each of the Directors entitled to vote. Such written consents may be executed in two or more counterparts, each of which shall be deemed an original and all of which, taken together, shall constitute one and the same document. Any such written consent shall be inserted in the minute book as if it were the minutes of a Board meeting. For purposes of these Bylaws, “executed” means: (a) writing that is signed; or (b)  an email transmission with sufficient information to determine the sender's identity or a Discord message or vote in an official Bevy channel or private chat where all maintainers are present."
```
Comment (BD103) [inline content/foundation/bylaws/index.md:104]:
```suggestion
Any action which could be taken at a meeting of the Board may be taken without a meeting if a written consent setting forth the action so taken is executed by each of the Directors entitled to vote. Such written consents may be executed in two or more counterparts, each of which shall be deemed an original and all of which, taken together, shall constitute one and the same document. Any such written consent shall be inserted in the minute book as if it were the minutes of a Board meeting. For purposes of these Bylaws, “executed” means: (a) writing that is signed; or (b) an email transmission with sufficient information to determine the sender's identity or a Discord message or vote in an official Bevy channel or private chat where all maintainers are present."
```

Just an extra space :)

File: content/foundation/meeting-minutes/actions-without-meeting/index.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+**Proposal:** So the proposal is: increase Alice's monthly payments to $6,417.67 CAD ($4,705.28 USD) as soon as Thera can make it possible. Vote on this comment with 👍  or 👎 to officially approve this proposal. Feel free to discuss first if you want to amend the proposal / notice any discrepancies.
+
+**Votes:** Alice I. Cecile abstains. All others voted yes.
+
+## 2024-07-16: Deprecating legacy sponsors
+
+![Deprecating legacy sponsors](deprecating_legacy_sponsors.png)
+
+**Proposal:**  Also: I think it is time to deprecate the "legacy sponsors" section. I've prepped [a branch](https://github.com/bevyengine/bevy-website/pull/1564) but I won't create the PR until we agree
```
Comment (BD103) [inline content/foundation/meeting-minutes/actions-without-meeting/index.md:133]:
```suggestion
**Proposal:** Also: I think it is time to deprecate the "legacy sponsors" section. I've prepped [a branch](https://github.com/bevyengine/bevy-website/pull/1564) but I won't create the PR until we agree
```

**Review by mockersf — APPROVED** [review]

File: content/foundation/meeting-minutes/actions-without-meeting/index.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+![Initial job titles](initial_job_titles.png)
+
+**Proposal:** All in favor of Project Lead (me once we hire me), Staff Engineer (soon to be Alice + future maintainer hires), and Principal for SME hires?
+
+**Proposed by:** Carter Anderson
+
+**Votes:** Unanamious yes
+
+## 2024-04-18: Usage of the Bevy logo by the unofficial Bevy Playfround
```
Comment (cart) [inline content/foundation/meeting-minutes/actions-without-meeting/index.md:42]:
```suggestion
## 2024-04-18: Usage of the Bevy logo by the unofficial Bevy Playground
```

**Review by cart — APPROVED** [review]

**Pull request merged**